contracts/DistributionCreator.sol

```diff
@@ -514,6 +520,20 @@ contract DistributionCreator is UUPSHelper, ReentrancyGuardUpgradeable {
         extensiveParams.rewardTokenDecimals = IERC20Metadata(distribution.rewardToken).decimals();
     }
 
+    /// @notice Tries to fetch extra data about the parameters in a distribution
+    function tryGetExtensiveDistributionParameters(
+        DistributionParameters memory distribution
+    ) external returns (bool success, ExtensiveDistributionParameters memory extensiveParams) {
+        (bool callSuccess, bytes memory returndata) = address(this).delegatecall(
```

Comment:
Ok smart to have a return value as a byte, but why wouldn't we do that on the atual failing code `globalState`/`fee`?
Because here if this one fail we don't even consider the distribution, while we just don't have the fee which is not essential for merkl just for the front